### ISSUE TITLE
Should have protocols (show where edges come from)

### ISSUE THREAD

**Issue by tedawson** [tedawson/Networks-of-others#1]
I think we should include some sort of text file showing when interactions occur in the film (justifying edges). e.g.:

4:08 Hempf speaks to Wiesler

Maybe this wouldn't need to include time, cause that would take forever... I have this for the first part written out by hand already, but need to come up with a consistent way to show relationships. Also, need to do the second half of the film...  

If we decided to get fancy, we could write a script to pull from the protocols onto a csv of edges.

**Comment by msheehan-DH** [comment]
This might be a good question for Mickey and Bobby. What kind of data do we need to create and deliver to justify our graphs (and so others could recreate our graphs from the data)? 

I have access to some student workers who might be able to do more of the (cough) grunt work. One computer science student is working to create OCR files from Fraktur (no guarantee this project will actually function). I can also get him to start cataloguing all these interactions. He could flag questionable instances for our review. I can look into that option...

**Comment by tedawson** [comment]
Agreed, @casadm and @bobbysmiley will probably be able to give some guidance on what data should be made available and how. I'll go ahead and invite them to collaborate on this repo.

Student workers doing grunt work sounds generally awesome... I think the actual cataloging of interactions from human readable format to CSVs can now be done by this critter I'm calling "[Gephicate](https://github.com/tedawson/gephicate)" (trying to decide whether to have the readme written in scatological language), but I'm sure we can find stuff for undergrads to do... possibly including watching the film and marking interactions, or at least proofing our work on that front.

Let me know how the Fraktur OCR thing goes. Sounds fun.
